insights/parsers/nova_api_log.py

```diff
@@ -1,17 +1,26 @@
+"""
```

Comment:
The "nova_api_log" is included in "https://github.com/RedHatInsights/insights-core/pull/188", better remove it from this PR.